connection_restore: Connection restored data was synced